gradle-plugins/src/integTest/resources/ivy/modern/gradle7/expectedGrandparentIvyDescriptorContents.txt

```diff
@@ -0,0 +1,22 @@
+  <configurations>
```

Comment:
What is different about the gradle7 version of these descriptors, vs. the existing ones under `resources/ivy/modern/expected*` ?